bucket/keepass-plugin-sequencer.json

```diff
@@ -0,0 +1,27 @@
+{
+    "homepage": "https://github.com/fireout/keepasssequencer",
+    "description": "Custom password generator algorithm plugin for KeePass 2.x to generate word sequences.",
+    "notes": "Default options located in \"$dir\\options.xml\".",
+    "version": "0.1.1",
+    "url": [
+        "https://github.com/fireout/keepasssequencer/releases/download/release%2F0.1.1/Sequencer.dll",
+        "https://github.com/fireout/keepasssequencer/releases/download/release%2F0.1.1/Sequencer.dll.config",
+        "https://github.com/fireout/keepasssequencer/releases/download/release%2F0.1.1/options.xml"
+    ],
+    "hash": [
+        "507cc90aa1c47cf213775c7fdb2eb39c8f4d34e9edab98d524dd2d9493b79661",
+        "46935a8e2f420c629001fb6fb2850e3dd35c0ab430a5e506d159dcab1090ca0a",
+        "80142a27be9faed4c8f3dad99bc77d2c196af07e6b8d57f7ccc1d831b971164d"
+    ],
+    "depends": "extras/keepass",
+    "installer": {
+        "script": [
+            "$pluginDir = New-Item -ItemType Directory \"$(appdir keepass)\\current\\Plugins\\Sequencer\"",
```

Comment:
Because it's not an `.plgx` plugin, when you install or uninstall, it can be locked by KeePass, if KeePass is running. Should it fail to install/uninstall in this case? Because now `$ErrorActionPreference` is **Continue** and script doesn't fail on error.

Reply:
Perhaps report to the user the failure, and that a reboot is required, and then mark the file for deletion on the next reboot?

Reply:
@rasa Maybe fail installation and report that file is locked and user should exit KeePass before installing? It's easier to do.
And add this information in notes and user will see it in `scoop info`.

Reply:
@ebugusey yeah, that sounds fine, whatever is easiest.